sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/plugin/KotlinRulesDefinition.java

```diff
@@ -35,10 +36,15 @@ public void define(Context context) {
       .createRepository(KotlinPlugin.KOTLIN_REPOSITORY_KEY, KotlinPlugin.KOTLIN_LANGUAGE_KEY)
       .setName(KotlinPlugin.REPOSITORY_NAME);
     RuleMetadataLoader ruleMetadataLoader = new RuleMetadataLoader(RESOURCE_FOLDER);
-// TODO: Add logic for rules that require language specific configuration at construction time
-    ArrayList<Class> checks = new ArrayList<>(CommonCheckList.kotlinChecks());
+
+    ArrayList<Class> checks = new ArrayList<>(CheckList.kotlinChecks());
     checks.add(CommentedCodeCheck.class);
     ruleMetadataLoader.addRulesByAnnotatedClass(repository, checks);
+
+    repository.rule("S100")
```

Comment:
I think test is missing for default value

Reply:
Added test for default value of this rule in Kotlin